src/codeu/chat/server/Model.java

```diff
@@ -68,9 +74,95 @@ public int compare(Time a, Time b) {
   private final Uuid.Generator userGenerations = new LinearUuidGenerator(null, 1, Integer.MAX_VALUE);
   private Uuid currentUserGeneration = userGenerations.make();
 
+  private final String database = "codeutest.db";
+  private final String userTable = "USER";
+  private final String sqliteClass = "org.sqlite.JDBC";
+
+  public Model () {
+    Connection conn = null; // Placeholder for connection to database
+    Statement stmt = null; // Placeholder for sql command
+    try {
+
+      // Loads the JDBC class dynamically to handle the SQL connection
+      // DriverManager cannot create the connection if this class isn't loaded at runtime
+      Class.forName(sqliteClass);
+      
+      // Connect to database. Creates database if it doesn't exist.
+      conn = DriverManager.getConnection("jdbc:sqlite:" + database);
+      DatabaseMetaData dbm = conn.getMetaData();
+      ResultSet rs = dbm.getTables(null, null, userTable, null);
+       
+      if (rs.next()) { // Table exists
+       
+        String sql = "SELECT ID, TIME, NAME FROM " + userTable;
+        stmt = conn.createStatement();
+        ResultSet users = stmt.executeQuery(sql);
+        // loop through the result set
+        while (users.next()) {
+          
+          String username = users.getString("NAME");
+          String uuidString = users.getString("ID");
+          uuidString = uuidString.substring(6, uuidString.length() - 1);
+          
+          try {
+            Uuid useruuid = Uuid.parse(uuidString);
+            User u = new User(useruuid, username, Time.fromMs(users.getLong("TIME"))); 
+            restoreUser(u);
+          } catch (IOException io) {
+              System.out.println(io.getMessage());
+          }
+        }
+
+        stmt.close();
+        conn.close();  
+      } else { // Table does not exist
+          stmt = conn.createStatement();
+          // Command to create USER table
+          String sql = "CREATE TABLE   " + userTable +
+          "(ID TEXT PRIMARY KEY     NOT NULL," +
+          " TIME           BIGINT     NOT NULL, " + 
+          " NAME           TEXT    NOT NULL" + 
+          ")"; 
+          stmt.executeUpdate(sql);
+          stmt.close();
+          conn.close();
+        }
+    } catch ( Exception e ) {
+      System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+      System.exit(0);
+    }
+}
+
+  public void executeSQL(Connection conn, String statement) {
```

Comment:
The body of this method is empty. Are you planning on implementing it? If so, please add a TODO with an explanation of what this method will do and how it will be used (or add the implementation). If not, you should remove this from the codebase.